package/MDAnalysis/guesser/DefaultGuesser.py

```diff
@@ -0,0 +1,79 @@
+from .base import GuesserBase
+import numpy as np
+import warnings
+from . import tables
+
+
+class DefaultGuesser(GuesserBase):
+    context = 'default'
+
+    def __init__(self):
+        self._guess = {'mass': self.guess_masses}
```

Comment:
Hmm. Instead of this, would it be tidier to have a class-level `_guess` dictionary like `TopologyAttr.transplants`? I'm not sure it makes sense for instances of guesser classes to differ.

Reply:
I'll check this and see how I could integrate the transplant idea